doc/pages/source/documentation/index.md

```diff
@@ -950,7 +950,27 @@ to the chain of middleware execution.
 ### Sub-plans
 
 -   *when to use?*
--   *how to use?*
+
+To use sub-plans, you must include the `Dynflow::Action::WithSubPlans` module
+and override the `create_sub_plans` method. Inside the `create_sub_plans`
+method, you use the `trigger` method to create sub-tasks that will be executed
+in no particular order during the run phase. These sub-tasks will not block the
+parent task.
```

Comment:
This is not precise: the parent task will still wait for the sub-plans to finish. The important thing is it's not blocking a thread in a pool while waiting. Mentioning that the reason for this is busy-waiting for the task in run method could lead to dead-locks, when all threads would be consumed by such a waiting, no tasks could be finished.